### PR TITLE
Use cypress helpers from `@nextcloud/cypress`

### PR DESCRIPTION
**Pull request by susnux**
### 📝 Summary
* Resolves only partly #2201 

Use user management commands from `@nextcloud/cypress` instead maintaining them by our self.

### 🏁 Checklist

- [x] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [x] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits
- [x] [Tests](https://github.com/nextcloud/text#-testing-the-app) (unit, integration and/or end-to-end) passing and the changes are covered with tests
- [x] Documentation ([README](https://github.com/nextcloud/text/blob/master/README.md) or [documentation](https://github.com/nextcloud/documentation/blob/master/admin_manual/configuration_server/text_configuration.rst#L2)) has been updated or is not required

### CODE REVIEW

File: cypress/README.md [hunk 1]
```diff
@@ -35,10 +33,9 @@ cy.getContent()
 
 ### Commands from `@nextcloud/cypress`
 
-We also use some commands from `@nextcloud/cypress`:
+We also use some commands from [`@nextcloud/cypress`](https://github.com/nextcloud/nextcloud-cypress):
 * `login(user)`
 * `logout()`
 * `createRandomUser().then(user => login(user))`
 * `createUser(user)`
```
Comment (max-nextcloud) [inline cypress/README.md:40]: Could you add the two commands here?
I think we should either keep this list complete or remove it entirely.
```suggestion
We also use some commands from [`@nextcloud/cypress`](https://github.com/nextcloud/nextcloud-cypress):
* `login(user)`
* `logout()`
* `createRandomUser().then(user => login(user))`
* `createUser(user)`
* `modifyUser(user, key, value)`
* `deleteUser(user)`
```

**Review by max-nextcloud — APPROVED** [review]
Code looks good. I suggest adding the commands to the list in the docs.

File: cypress/README.md (hunk 1 above)
Comment (susnux) [reply cypress/README.md:40]: I would vote for removing and only keep a reference of our own commands + a note about the nextcloud package.
(so keep the table but remove the list and only keep `We also use some commands from [`@nextcloud/cypress`](https://github.com/nextcloud/nextcloud-cypress).`

Comment (max-nextcloud) [reply cypress/README.md:40]: Fine with me.

**Pull request merged**